lib/ain-ocean/src/api/pool_pair/mod.rs

```diff
@@ -468,11 +452,20 @@ async fn list_pool_swaps_verbose(
             Ok((k, _)) => k.0 == id,
             _ => true,
         })
-        .map(|item| {
+        .map(|item| async {
             let (_, swap) = item?;
-            Ok(PoolSwapVerboseResponse::from(swap))
+            let from_to =
+                find_swap_from_to(&ctx, swap.block.height, swap.txid, swap.txno.try_into()?)
```

Comment:
This is expensive and should only be done if we haven't indexed the `TxResult::PoolSwap`.
Should make the `TxResult::PoolSwap` optional in the PoolSwap `Index` implementation

Reply:
the poolswap index impl is interesting.. checking...

Reply:
if its not mistaken.. only need to execute the `TxResult::PoolSwap` with ocean enable to reduce op cost
[38149e8](https://github.com/DeFiCh/ain/pull/2926/commits/38149e8b20e0a745dfa31a8c9c6b9be71b71d0c5)

Reply:
https://github.com/DeFiCh/ain/blob/38149e8b20e0a745dfa31a8c9c6b9be71b71d0c5/lib/ain-ocean/src/indexer/pool.rs#L39-L41
another blocker for deployment.. fixing it at another PR